recipe/meta.yaml

```diff
@@ -22,14 +22,18 @@ build:
   skip: true # [gpu_variant == "cuda-11"]
   # do not mix cublas and mkl/openblas
   skip: true # [((gpu_variant or "").startswith('cuda') and (blas_impl != "cublas")) or (not (gpu_variant or "").startswith('cuda') and (blas_impl == "cublas"))]
+  # do not mix CPU optimization with cublas
+  skip: true # [x86_64_opt == "v3" and gpu_variant != "none"]
```

Comment:
What's this for?

Reply:
For the CUDA/GPU variants of the package no CPU optimization variant is produced, these packages all have the default CPU support. 

The build matrix could be expanded if this is desired so there are two packages for each CUDA variant, one with compatibility and one with the v3 CPU optimizations.

Reply:
Is there a downside for enabling CPU optimizations and GPU at the same time? E.g. would this be the "best possible" optimized build?

Reply:
CPU optimizations were enabled for CUDA variants in #8